spec/requests/shopping_list_items_spec.rb

```diff
@@ -169,7 +255,7 @@
 
         it 'does not return content' do
           create_item
-          expect(response.body).to be_empty
+          expect(response.body).to be_blank
```

Comment:
Come to find out, `empty?` isn't defined on `nil`, so this would blow up if the body were `nil` when really we'd be OK with that.